docs/source/concepts/local-ledger.rst

```diff
@@ -343,8 +344,14 @@ Local ledgers
 As explained in the Daml Ledger Model, parties see only a :ref:`projection <da-model-projections>` of the shared ledger for privacy reasons.
 Like consistency, projection extends to causality graphs as follows.
 
+Definition »Stakeholder informee«
+  A party `P` is a **stakeholder informee** of an action `act` if `P` is an informee of `act` and, if `act` is an action on a contract then `P` is a stakeholder of the contract.
+
+An **Exercise** and **Fetch** action acts on the input contract, a **Create** action on the created contract, and a **NoSuchKey** action does not act on a contract.
+So for a **NoSuchKey** action, the stakeholder informees are the key maintainers.
+
 Definition »Causal consistency for a party«
-  A causality graph `G` is **consistent for a party** `P` (`P`-consistent) if `G` is consistent on all the actions that `P` is an informee of.
+  A causality graph `G` is **consistent for a party** `P` (`P`\ -consistent) if `G` is consistent on all the actions that `P` is a stakeholder informee of.
 
 The notions of `X`\ -minimality and `X`\ -reduction extend to parties accordingly.
 
```

Comment:
... is consistent for the Bank because the Bank is a **stakeholder** informee of exactly the highlighted actions ... ?

Reply:
(this bit is a couple lines below where I can comment)

Reply:
Thanks for pointing this out. Changed.